packages/page-pattern-modal/src/components/page-pattern-modal.tsx

```diff
@@ -313,7 +314,7 @@ class PagePatternModal extends Component< PagePatternModalProps, PagePatternModa
 							id={ `page-pattern-modal__heading-${ instanceId }` }
 							className="page-pattern-modal__heading"
 						>
-							{ __( 'Add a page', __i18n_text_domain__ ) }
+							{ this.props.title || __( 'Add a page', __i18n_text_domain__ ) }
```

Comment:
I think there was some discussion in the p2 about changing the description as well as the title? Feels like "start with a blank page" doesn't feel right since they just asked to do that during onboarding. Maybe "continue with blank page"?

Reply:
Yes, you're right. Now I see in p9Jlb4-2qt-p2#comment-3874

> The subtitle also needs adjustment

@ollierozdarz, WDYT? Should we go with "Pick a pre-defined layout or continue with a blank page" as subtitle?

Reply:
Apologies @razvanpapadopol I missed this! Great suggestion @p-jackson , **continue** here makes complete sense and feels like a good progression.

Let's roll with the suggestion, "Pick a pre-defined layout or continue with a blank page".

Reply:
Done! Here's how it looks together with the smaller font for the title as suggested by @olaolusoga 

<img width="1277" alt="Screenshot 2021-05-12 at 12 36 57" src="https://user-images.githubusercontent.com/14192054/117954119-5f59a280-b31f-11eb-8b60-8f8b55803d8e.png">